components/ActionSheet/examples/ActionSheetTypicalUseExampleViewController.m

```diff
@@ -100,7 +100,7 @@ + (NSDictionary *)catalogMetadata {
   return @{
     @"breadcrumbs" : @[ @"Action Sheet", @"Action Sheet" ],
     @"primaryDemo" : @YES,
-    @"presentable" : @NO
+    @"presentable" : @YES,
```

Comment:
Should this be changed to presentable?

Reply:
Yes, so that we can test it within the catalog.